src/ja/user/cli.py

```diff
@@ -0,0 +1,6 @@
+class UserClientCLIHandler:
+    """
+    Class for handling user input on the user client. Transforms raw user input
```

Comment:
Might make sense to clarify what "raw user input" is. When I hear it, I think about the "raw_input()" thing in python, but here you actually mean command line arguments, right?

Reply:
Correct.
By "raw user input" I mean command line arguments.
It's "raw" input in the sense that it has not been processed yet and is therefore not directly usable.
Do you think "unprocessed user input" would be a better term?

Reply:
Why not just use "command line arguments"? Do we expect any other form of input?

Reply:
Or maybe command line input.